README.md

```diff
@@ -127,9 +138,18 @@ If any of the AWS API calls outrightly fail, `error` is emitted. If you don't ha
 ### gotMessages {number}
 Emitted when Squiss asks SQS for a new batch of messages, and gets some (or one). Supplies the number of retrieved messages.
 
+### handled {Message}
+Emitted when a message is handled by any means: deleting, releasing, or calling `keep()` or `handledMessage()` on it. 
+
 ### queueEmpty
 Emitted when Squiss asks SQS for new messages, and doesn't get any.
 
+### released {Message}
+Emitted after `release()` or `releaseMessage` has been called and the VisibilityTimeout of a message has successfully been changed to `0`. The `handled` event will also be fired for released messages, but that will come earlier, when the release function is initially called.
```

Comment:
Similar to the deleted question... it sounds like here you could call `.release()` and the `handled` event would emit, but the actual `released` event wouldn't happen until the VisibilityTimeout gets set to 0 which is what (I believe) really lets the message be released -- so anything that could fail here to where the message wouldn't get released and cause issues since it was already marked as handled?!